tests/avifgridapitest.c

```diff
@@ -117,11 +137,11 @@ static avifBool decode(const avifRWData * encodedAvif)
 //------------------------------------------------------------------------------
 
 // Generates, encodes then decodes a grid image.
-static avifBool encodeDecode(int columns, int rows, int cellWidth, int cellHeight, avifPixelFormat yuvFormat, int expected_success)
+static avifBool encodeDecode(int columns, int rows, int cellWidth, int cellHeight, int depth, avifPixelFormat yuvFormat, int createAlpha, int expected_success)
```

Comment:
Please also declare `expected_success` as `avifBool` and rename it `expectedSuccess`. See https://github.com/AOMediaCodec/libavif/pull/857/commits/5d8075e258cf412facca823a728d5baaf79eb8d5.

Reply:
Sorry about that, I mixed up libwebp and libavif code styles.

Thanks for the commit example. Next time feel free to submit trivial fixes.